redbot/cogs/mod/kickban.py

```diff
@@ -334,6 +334,15 @@ def remove_processed(ids):
                         user_id=user_id
                     )
 
+        member = namedtuple("Member", "id guild")
+        async with self.settings.guild(guild).current_tempbans() as guild_tempbans:
```

Comment:
```suggestion
        async with self.config.guild(guild).current_tempbans() as guild_tempbans:
```

Please ensure attributes you are using exist.
`self.settings` has been renamed tgo `self.config` a few releases ago.